R/statistic-glm.R

```diff
@@ -195,16 +195,16 @@ dpGLM <- setRefClass(
 )
 
 dpGLM$methods(
-    initialize = function(mechanism, var.type, n, rng, formula, objective, epsilon=NULL,
+    initialize = function(mechanism, var.type, n, rng=NULL, formula, objective, epsilon=NULL,
```

Comment:
good question. Would a user use glm on a logical variable? Because the only reason the rng defaults to NULL is because entering rng for a logical variable is optional.

Reply:
yes, user could enter logical variable, so range is optional. If range is necessary, the call the checkrange() checks for non-null range.